nca/FWRules/ConnectivityGraph.py

```diff
@@ -91,6 +100,36 @@ def _get_peer_color(is_livesim, is_ip_block):
             return "red2"
         return "blue"
 
+    def get_connections_without_fw_rules_txt_format(self):
+        """
+        :rtype: str
+        :return: a string of the original peers connectivity graph content (without adding fw-rules or livesim peers)
+        """
+        lines = set()
+        workload_name_to_peers_map = {}  # a dict from workload_name to pods set, to track replicas and copies
+        for connections, peer_pairs in self.connections_to_peers.items():
+            for src_peer, dst_peer in peer_pairs:
+                src_peer_name, _ = self._get_peer_name(src_peer, True)
+                if str(connections) == 'All connections' and src_peer == dst_peer:  # relevant with all connections only
+                    # add the pod to the map with its workload name
+                    if src_peer_name not in workload_name_to_peers_map:
+                        workload_name_to_peers_map[src_peer_name] = set()
+                    workload_name_to_peers_map[src_peer_name].add(src_peer)
+                    continue  # after having the full dict, lines from pod to itself will be added for workload names
+                    # with only one pod.
+                    # if a peer has different replicas or copies, a connection from it to itself will be added automatically
+                    # only if there are connections between the replicas too (not only from a single pod to itself)
+                dst_peer_name, _ = self._get_peer_name(dst_peer, True)
+                conn_str = connections.get_simplified_connections_representation(True)
+                conn_str = conn_str.title() if not conn_str.isupper() else conn_str
```

Comment:
what is `conn_str.title()` ?
what kind of update to `conn_str` is made on this line and why?

Reply:
it replaces `All connections` with ` All Connections` (second is used in netpol analyzer)